doc/admin-guide/files/records.config.en.rst

```diff
@@ -22,7 +22,7 @@
 records.config
 **************
 
-The :file:`records.config` file (by default (:ts:cv:`proxy.config.config_dir`), located in
+The :file:`records.config` file (by default, located in
```

Comment:
This should mention `TS_ROOT` which was originally described in the configuration variable description. Effectively that text should be moved here.

Reply:
Resolved.